### PR TITLE
fixing addon dependency for other addons

### PR DESCRIPTION
**Pull request by vesper2000**
#minor#

### CODE REVIEW

File: blueprints/ember-frost-object-browser/index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+    @returns {Promise} package names and versions
+  */
+  afterInstall: function () {
+    return this.addAddonsToProject({
+      packages: [
+        {name: 'ember-frost-theme', target: '^1.3.2'},
+        {name: 'ember-frost-bunsen', target: '^2.2.0'},
+        {name: 'ember-frost-button', target: '^1.4.2'},
+        {name: 'ember-frost-checkbox', target: '^1.2.0'},
```
Comment (sglanzer) [inline blueprints/ember-frost-object-browser/index.js]: Where is checkbox used?  If it's just in the form then it should come from bunsen

Comment (vesper2000) [inline blueprints/ember-frost-object-browser/index.js]: It is used here:
image

Comment (sglanzer) [inline blueprints/ember-frost-object-browser/index.js]: Urgh, ok - this component is a bit of a mess but I won't hold up the PR

**Comment by sglanzer** [comment]
Approved

File: blueprints/ember-frost-object-browser/index.js (hunk 1 above)
Comment (vesper2000) [inline blueprints/ember-frost-object-browser/index.js]: The build timed out on the test part :(  so will need to sort that out.

**Pull request merged**